### DISCUSSION TITLE
The Transparent Rendering android sample doesn't work when upgrading to Filament 1.35.0

### DISCUSSION THREAD

**Discussion by ndming** [google/filament#6919] [Q&A]
When upgrading from release `1.34.0` to `1.35.0`, the [transparent-rendering](https://github.com/google/filament/tree/main/android/samples/sample-transparent-view) android sample no longer shows a transparent background but instead a black one. I have included some captures below.

The problem seems to persist even in the latest release (`1.38.0`). How can I fix this?

| `1.34.0` | `1.35.0` | 
|---         |---         |
|![image](https://github.com/google/filament/assets/61007865/1f3088dc-23e2-44a8-a431-fe42e0ce24fc)|![image](https://github.com/google/filament/assets/61007865/7d42586d-39f8-4561-9d1f-e3df96610829)|

**Comment by romainguy** [comment]
Thanks for reporting this. Looks like a bug, hopefully with the sample itself.

@bejado @poweifeng Any ideas about recent changes that caused this?

**Comment by ndming** [comment]
The bug might have something to do with `DynamicResolutionOptions`. I notice before version `1.35.0`, if I configure the `view` like below, the problem also appears, though it may take a while to see the opaque background showing up (on my pretty old Android device the bug shows up immediately, while on the Pixel 6 Pro API 33 emulator, it appears after a long while if I also enable MSAA, FXAA, ambient occlusion, and bloom).

```kotlin
view.dynamicResolutionOptions = view.dynamicResolutionOptions.apply {
    enabled = true
    quality = View.QualityLevel.MEDIUM
}
```

If I disable `View.DynamicResolutionOptions`, or set the `QualityLevel` to `QualityLevel.LOW`, everything's back to normal. However, going to `1.35.0`, these configurations don't affect anything, it's opaque black no matter whether I disable dynamic resolution or not. 

Hope these observations help narrow down the cause.

**Reply by bejado** [reply]
I just tried to repo on the tip on main, but can't seem to reproduce (even after waiting several minutes):

![Screenshot_20230626-144930](https://github.com/google/filament/assets/5298046/3455d46c-d2a1-4761-bbcd-a8b83aa01450)

I'm using a Pixel 4a to test. Maybe this is a device-specific issue? What old Android device are you using?

**Reply by ndming** [reply]
I was using the Samsung J7 Prime (Android 8.1 API 27)

**Comment by sebapian** [comment]
Hi there, I think I am facing a similar issue possibly with the same version update (v1.34.0 to v1.35.0++). Opened a discussion here:
https://github.com/google/filament/discussions/7220
